pre_commit/commands/try_repo.py

```diff
@@ -23,7 +23,7 @@ def try_repo(args):
         if args.hook:
             hooks = [{'id': args.hook}]
         else:
-            repo_path = Store(tempdir).clone(args.repo, ref)
+            repo_path = Store(tempdir).clone({'repo': args.repo, 'sha': ref})
```

Comment:
I don't understand the benefit of changing from two position arguments to a dictionary. Can you explain?

Reply:
i would have needed three args, and all came from the dict. And at some point, since both existing args were contained in the dict, it made sense to just pull from the dict as the only arg.

Reply:
Ah yes, I skipped over the `repo_config` when reading thru it: `allowed_git_envs = repo_config.get('git_env', [])`

That makes sense.